src/query/query_executor.h

```diff
@@ -967,6 +967,7 @@ struct xasl_node
   const char *query_alias;
   int dbval_cnt;		/* number of host variables in this XASL */
   bool iscan_oid_order;
+  int max_iterations;
```

Comment:
please add a blank line between line 968 and 969 to separate xasl_cache entries from others.